FordLabs/peoplemover
# Remove MatomoService from the codebase

## Issue
Resolves N/A

## What was done
- [x] Remove MatomoService from the codebase

## How to test
1. Ensure that there is nothing related to matomo in the codebase anymore
2. Ensure the app's functionality has not changed at all
